k8s/models/job.py

```diff
@@ -36,7 +36,12 @@ class JobSpec(Model):
     parallelism = Field(int)
     selector = Field(LabelSelector)
 
+    
```

Comment:
Codacy found an issue: [blank line contains whitespace (W293)](https://app.codacy.com/manual/fiaas/k8s/pullRequest?prid=5919885)

Reply:
Not sure why it complains about it. I have removed and added the lines nothing should be there.